spring-cloud-kubernetes-archaius/src/main/java/org/springframework/cloud/kubernetes/archaius/ArchaiusConfigMapSourceConfiguration.java

```diff
@@ -26,6 +25,8 @@
 import io.fabric8.kubernetes.client.KubernetesClientException;
 import io.fabric8.kubernetes.client.Watch;
 import io.fabric8.kubernetes.client.Watcher;
+import io.fabric8.kubernetes.client.utils.Utils;
```

Comment:
I'd prefer to use Spring classes except when needed. `StringUtils.isEmpty()` or `!StringUtils.hasText()`.